package.json

```diff
@@ -58,10 +59,11 @@
     "gulp-load-plugins": "^1.2.4",
     "gulp-util": "^3.0.7",
     "gzip-js": "~0.3.2",
-    "jest": "^15.1.1",
-    "jest-config": "^15.1.1",
-    "jest-jasmine2": "^15.1.1",
-    "jest-runtime": "^15.1.1",
+    "jest": "^17.0.2",
+    "jest-cli": "17.0.2",
```

Comment:
`jest` just exports `jest-cli` (https://github.com/facebook/jest/blob/master/packages/jest/src/jest.js) so only one of them would be enough here.